ui/src/components/Feed/FeedPost.svelte

```diff
@@ -58,30 +56,35 @@
       return api.portal.do.subscribe(key);
     }
 
-    // recursively fetch replies so we can display the total number of replies per OP
-    const getNestedReplies = (replies) => {
-      if (Array.isArray(replies) && replies.length > 0) {
-        for (let reply of replies) {
-          if (!allReplies.has(reply)) {
-            allReplies.add(reply);
-            const repliesToReply = getReplies(reply);
-            if (repliesToReply) {
-              getNestedReplies(repliesToReply);
-            }
-          }
-        }
-      }
-    };
-
-    getNestedReplies(getReplies(key));
-
     let likes = [...(getLikes(key.ship, key) || [])];
 
     numLikes = likes.length;
     if (isLikedByMe && !likes.find((l) => l.ship === me)) numLikes++;
     isLikedByMe = isLikedByMe || !!likes.find((l) => l.ship === me);
   };
 
+  // recursively fetch replies so we can display the total number of replies per OP
+  const getNestedReplies = async (replies) => {
+    if (Array.isArray(replies) && replies.length > 0) {
+      for (let reply of replies) {
+        if (!allReplies.has(reply)) {
+          allReplies.add(reply);
+          const repliesToReply = getIndexerAndLocalReplies(reply);
+          if (repliesToReply) {
+            await getNestedReplies(repliesToReply);
+          }
+        }
+      }
+    }
+    allReplies = allReplies;
```

Comment:
the other quirk is that svelte doesn't see a change in the set variable as a result of `.add()`, so you have to reassign the variable to itself to get the compiler to recognise that the value is changing as a result of this function. stupid? yes.